src/kOS/Binding/MissionSettings.cs

```diff
@@ -60,36 +60,24 @@ public override void AddTo(SharedObjects shared)
 
             shared.BindingMgr.AddGetter("TARGET", () =>
             {
-                if (shared.Vessel != FlightGlobals.ActiveVessel)
-                {
-                    throw new kOS.Safe.Exceptions.KOSSituationallyInvalidException("TARGET can only be returned for the Active Vessel");
-                }
-                var currentTarget = FlightGlobals.fetch.VesselTarget;
+                var target = shared.Vessel == FlightGlobals.ActiveVessel ?
+                    FlightGlobals.fetch.VesselTarget : shared.Vessel.targetObject;
 
```

Comment:
It would probably work with just `shared.Vessel.targetObject` but I wanted to be safe and preserve the original without any change.